pwndbg/wrappers/file.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.wrappers
+
+cmd_name = "file"
+
+@pwndbg.wrappers.OnlyWithCommand(cmd_name)
+def is_statically_linked():
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cmd = [is_statically_linked.cmd_path, local_path]
+
+    file_out = pwndbg.wrappers.call_cmd(cmd)
+
+    if "statically" in file_out:
```

Comment:
This will give us wrong results if the binary name or its interpreter path contains `statically`.

```bash
$ gcc main.c -o not_static
$ gcc main.c -static -o static
$ file *static
not_static: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=2d7bfb5e54f54cf1a86be713686103db0c91d0af, not stripped
static:     ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, for GNU/Linux 2.6.32, BuildID[sha1]=690a8e655d5c7752c003bb32f45f272dba8cebbd, not stripped
```